src/test/unit/lib/rules-engine/index.test.js

```diff
@@ -72,7 +72,10 @@ test('Evaluates a rule based on demo data', async (t) => {
 
     const input = {
         path: '/transfers',
-        body: '123',
+        // body: '123', // <-- This still passes and seems to be a bug with the json-rules-engine: https://github.com/CacheControl/json-rules-engine/releases/tag/v2.3.6. This comment is kept here until this can be addressed in future with the library dependency.
```

Comment:
We're using the latest version: https://github.com/mojaloop/mojaloop-simulator/blob/master/src/lib/rules-engine/package.json#L13

I think this is because we need to update the syntax of this expression to be:
```
path: '$.transfers',
body: '123',
method: 'POST'
```

See [this example](https://github.com/cachecontrol/json-rules-engine/blob/HEAD/examples/03-dynamic-facts.js#L31) of the update json-rules-engine syntax

Reply:
@lewisdaly I am not following you, I have updated all the paths to "$."?


Please see notes above:
```
fix(mojaloop/#2575): mojaloop-simulator rules are not executing as expected
- updated unit tests to cover issues for mojaloop/project#2575
- issue is caused by an update to the json-rules-engine. The path defined in rules must now start with "$" (e.g. $.<attribute>), previously .<attribute> would work but no longer functional due to the dependency being updated
- updated example rule paths with the missing $ prefix
```